db/schema.rb

```diff
@@ -11,11 +11,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160912232500) do
+ActiveRecord::Schema.define(version: 20160913145325) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "categories", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "categorizations", id: false, force: :cascade do |t|
```

Comment:
We struggled with the underscore table join last night / this morning. In particular we weren't getting the pluralization correct across all the various files / classes and we were getting errors. This passes the spec, but I agree it is broadly named. Perhaps we can take a second pass at this if anyone feels confident with rails g migration join tables and can troubleshoot the errors @susiirwin and I ran into last night. Alternatively, we can discuss better names for categorizations table. @tgisg